linux_os/guide/system/accounts/accounts-banners/gui_login_banner/dconf_gnome_login_banner_text/tests/wrong_value_stig.fail.sh

```diff
@@ -1,21 +1,30 @@
 #!/bin/bash
 # profiles = xccdf_org.ssgproject.content_profile_stig
-# packages = dconf,gdm
```

Comment:
Just making sure that this removal was intentional.

Reply:
yes, otherwise it will fail on Ubuntu, and also you have "install_dconf_and_gdm_if_needed" on line 12 to install them if needed